docs/CONTRIBUTING.md

```diff
@@ -23,6 +23,7 @@ Emoji | Description
 :sparkles: `:sparkles:` | When you applied clang-format.
 :globe_with_meridians: `:globe_with_meridians:` | When you worked on translations.
 :art: `:art:` | When you improved / added assets like themes.
+:lipstick: `:lipstick:` | When you worked on the UI.
```

Comment:
Good idea, but I would be a bit more specific as most parts of Fly-Pie are somehow UI :)

```suggestion
:lipstick: `:lipstick:` | When you worked on the UI of the preferences dialog.
```